src/Features/Core/Portable/SolutionCrawler/State/AbstractAnalyzerState.cs

```diff
@@ -61,7 +61,7 @@ public async Task<TData> TryGetExistingDataAsync(TValue value, CancellationToken
 
             try
             {
-                using var storage = await persistService.GetStorageAsync(solution, cancellationToken).ConfigureAwait(false);
+                await using var storage = await persistService.GetStorageAsync(solution, cancellationToken).ConfigureAwait(false);
```

Comment:
The dispose also needs to be configured in production code, except in cases where JTF is used.

```suggestion
                await using var storage = (await persistService.GetStorageAsync(solution, cancellationToken).ConfigureAwait(false)).ConfigureAwait(false);
```